src/main/java/be/jidoka/jdk/keycloak/admin/domain/User.java

```diff
@@ -59,12 +58,25 @@ public String getEmail() {
 		return email;
 	}
 
+	public boolean isEnabled() {
+		return enabled;
+	}
+
 	public String getPictureUrl() {
-		return pictureUrl;
+		return this.getSingleAttribute(PICTURE_URL_ATTRIBUTE)
+				.orElse(null);
 	}
 
-	public List<String> getClientRoles() {
-		return unmodifiableList(clientRoles);
+	public Set<String> getClientRoles() {
+		return unmodifiableSet(clientRoles);
+	}
+
+	public Optional<String> getSingleAttribute(String key) {
```

Comment:
Does this need to be public?

Reply:
Would be handy if a user of this lib gets a User back and he/she wants to get a custom value from the attributes of the User that there is already a utility method for this on the User object itself.

For us it would mean 
```
user.getSingleAttribute("tenantId").map(Long::valueOf)
```